docs/quickstart.md

```diff
@@ -526,7 +442,7 @@ spec:
   provisionPolicy: ProvisionNewCluster
 ```
 
-### Apply the app.yaml
+Then apply it:
 ```
 kubectl apply -f app.yaml
 ```
```

Comment:
FYI, `kubectl get managed` shows you all the managed resources that are present in the cluster if you'd like to avoid reference extractions from claims.

Reply:
Awesome, that sounds like a great enhancement, will add with a follow-on PR!